File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ClientSession.java
package ru.iqchannels.sdk.schema;


import androidx.annotation.Nullable;

/**
 * Copyright iqstore.ru. All Rights Reserved.
 */
public class ClientSession {
    public long Id;
    public long ClientId;
    public String Token;
    public boolean Integration;
    @Nullable public String IntegrationHash;
    @Nullable public String IntegrationCredentials;
    public long CreatedAt;
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ClientSession.kt
package ru.iqchannels.sdk.schema

class ClientSession {
	var Id: Long = 0
	var ClientId: Long = 0
	var Token: String? = null
	var Integration = false
	var IntegrationHash: String? = null
	var IntegrationCredentials: String? = null
	var CreatedAt: Long = 0
}
